monkey/infection_monkey/utils/load_plugins.py

```diff
@@ -0,0 +1,41 @@
+import importlib
+import inspect
+import logging
+from os.path import dirname, basename, isfile, join
+import glob
+
+LOG = logging.getLogger(__name__)
+
+
+def _get_candidate_files(base_package_file):
+    """
+    Gets all files under current directory(/actions)
+    :return: list of all files without .py ending
```

Comment:
If you don't mind fix my incorrect documentation or remove it altogether.

Reply:
I'm for removing it, simple function